Gem/Code/Source/Automation/ScriptReporter.h

```diff
@@ -88,6 +88,10 @@ namespace AtomSampleViewer
         // For exporting test results
         void ExportTestResults();
 
+        // Path to the exported test results file.
+        AZStd::string m_exportedTestResultsPath = "Click the 'Export Test Results' button.";
+        AZStd::string GetExportedTestResultsPath() const;
```

Comment:
When adding member functions and variables, it's important to determine its access of level. Best practice is to expose functions/variables on an as-needed basis. So essentially, only the ScriptReporter class needs to access these, so declaring them as public is too permissive. Please move them down below the private section. Lastly, be sure to group the member function and member variable separately (follow the formatting within this class)